wiki/Gameplay/Judgement/Notelock/en.md

```diff
@@ -30,3 +30,11 @@ Since objects in regular osu! beatmaps are meant to be cleared in chronological
 ## Prevention
 
 From a mapper's perspective, notelock could be prevented by carefully choosing the OD value of a [difficulty](/wiki/Beatmap/Difficulty) according to its object density and BPM. On beatmaps with BPM of 200 and higher, OD 5 and higher is recommended. For a more detailed breakdown, refer to the "[Avoiding notelock at high BPM](https://osu.ppy.sh/community/forums/topics/334458)" guide.
+
+## Changes to the mechanics
+
+With the release of osu!(lazer), there was a new notelock, which was not included in the stable version. The change is in the way it works: the first object stops locking the second object once the first object's time window reaches a offset of 0 ms.
```

Comment:
```suggestion
In comparison to osu!(stable), notelock was made more lenient in osu!(lazer) because the first hit object would stop locking the second hit object once the first hit object's time window reached an offset of 0 ms.
```

I don't have any accurate source on this, so I can't check if "0 ms" is correct.